dynatrace-operator/questions.yml

```diff
@@ -56,6 +56,13 @@ questions:
     type: string
     group: "Global Configuration"
 
+  - variable: customPullSecret
+    label: "Define a custom pull secret"
+    description: "Optional: Defines a custom pull secret in case you use a private registry when pulling images from the dynatrace environment"
```

Comment:
```suggestion
    description: "Optional: Defines a custom pull secret in case you use a private registry when pulling images from the Dynatrace environment"
```